Point-of-No-Return/Collision.cpp

```diff
@@ -0,0 +1,100 @@
+﻿#include "Collision.h"
+
+#include <vector>
+
+#include "Object.h"
+
+std::vector<Vec2> Collision::SearchBlock(const Vec2& character_pos,const Size& size,int** map)
```

Comment:
このファイルの中の関数群を
```
namespace Collision
{
}
```
の中に入れるべきですね。

そうすれば、例えばこの関数なら
```suggestion
std::vector<Vec2> SearchBlock(const Vec2& character_pos,const Size& size,int** map)
```
と書けるようになりますよ。

余談ですが、 [Google的にはnamespaceの中に入れてもインデント下げない](https://ttsuki.github.io/styleguide/cppguide.ja.html#Namespace_Formatting) らしいです…

Reply:
namespaceのインデントを下げないように修正しました。https://github.com/human-osaka-game-2019/Point-of-No-Return/pull/39/commits/8bc6f4515cf68f72672171a397d00def58bdc561

Reply:
指摘の本題はインデントではなく、「cppの中身をnamespaceで括るべきでは？」という話です。

Reply:
修正しました。https://github.com/human-osaka-game-2019/Point-of-No-Return/pull/39/commits/2f49a544f12267523e493ada7f5401f757254692